### PR TITLE
solved error of checked by piece at. It was showing the position in numeric and cahnged it into alphanumeric system

### PR DESCRIPTION
**Pull request by project-aps**

### CODE REVIEW

**Comment by project-aps** [comment]
merging to master after reviewing

**Review by project-aps — COMMENTED** [review]
merging to master

**Comment by shreeroshan** [comment]
merging to master

**Review by shreeroshan — COMMENTED** [review]
..

**Review by shreeroshan — COMMENTED** [review]
merging to master

File: game.h [hunk 1]
```diff
@@ -1571,12 +1571,17 @@ class Game
...
+        return (num + 96);
+    }
+
     /////////////////////////////////////////////////////////////////////////////////////////
     // it prints the piece location which gives the king check
     void print_check_by_piece_details()
     {
         cout << "CHECK BY:\n";
-        cout << "PIECE_AT: " << this->check_by_piece_x << this->check_by_piece_y << endl;
+        cout << "PIECE_AT: " << numericToAlphabet(this->check_by_piece_x) << this->check_by_piece_y << endl;
```
Comment (shreeroshan) [inline game.h:1584]: change the numeric key with alphabet

**Review by shreeroshan — COMMENTED** [review]
update..

**Review by shreeroshan — COMMENTED** [review]
Approved

File: game.h (hunk 1 above)
Comment (shreeroshan) [inline game.h]: update

**Review by shreeroshan — APPROVED** [review]
..

**Pull request merged**